buildSrc/src/main/kotlin/appPackage/Configurations.kt

```diff
@@ -0,0 +1,15 @@
+object Flavors {
+    const val PRODUCTION = "production"
+    const val STAGING = "staging"
+}
+
+object Module {
+    const val JSONAPI_CORE = ":jsonapi-kotlin:core"
```

Comment:
Then update the dependency. 
```suggestion
    const val JSONAPI_CORE = [":jsonapi-kotlin:core"](co.nimblehq.jsonapi:core:${Version.JSON_API})
```
We should move `JSONAPI_CORE` to `Dependency.kt` in this case.

Reply:
I updated it in 7fcdc128

![Screenshot 2023-04-25 at 15 47 56](https://user-images.githubusercontent.com/22606906/234224462-330a760f-c675-4d5b-8928-b98c41807056.png)